backend/database/database.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const host = process.env.host || 'localhost';
+
+const connection = () => mongoose.connect(`http://${host}:27017/data`, () => console.info(`database connected at ${host}`));
```

Comment:
this is on one line in my code. is this too much code to put on one line? i set up lint to require all my implicit returns to be on the same line rather than stepped down. just a small formatting thing.

Reply:
you should have a 150 character limit i think is the norm